tsl/test/expected/dist_policy.out

```diff
@@ -0,0 +1,163 @@
+-- Now simulate drop_chunks running automatically by calling it
+-- explicitly. Show chunks before and after.
+SELECT * FROM test.remote_exec(NULL, $$ SELECT show_chunks('conditions'); $$);
+NOTICE:  [dist_policy_data_node_1]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_1]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_1_chunk 
+_timescaledb_internal._dist_hyper_1_4_chunk 
+_timescaledb_internal._dist_hyper_1_7_chunk 
+_timescaledb_internal._dist_hyper_1_10_chunk
+_timescaledb_internal._dist_hyper_1_13_chunk
+(5 rows)
+
+
+NOTICE:  [dist_policy_data_node_2]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_2]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_2_chunk 
+_timescaledb_internal._dist_hyper_1_5_chunk 
+_timescaledb_internal._dist_hyper_1_8_chunk 
+_timescaledb_internal._dist_hyper_1_11_chunk
+_timescaledb_internal._dist_hyper_1_14_chunk
+(5 rows)
+
+
+NOTICE:  [dist_policy_data_node_3]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_3]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_3_chunk 
+_timescaledb_internal._dist_hyper_1_6_chunk 
+_timescaledb_internal._dist_hyper_1_9_chunk 
+_timescaledb_internal._dist_hyper_1_12_chunk
+_timescaledb_internal._dist_hyper_1_15_chunk
+(5 rows)
+
+
+ remote_exec 
+-------------
+ 
+(1 row)
+
+SELECT test_drop_chunks(:drop_chunks_job_id);
+ test_drop_chunks 
+------------------
+ 
+(1 row)
+
+SELECT * FROM test.remote_exec(NULL, $$ SELECT show_chunks('conditions'); $$);
+NOTICE:  [dist_policy_data_node_1]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_1]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_7_chunk 
+_timescaledb_internal._dist_hyper_1_10_chunk
+_timescaledb_internal._dist_hyper_1_13_chunk
+(3 rows)
+
+
+NOTICE:  [dist_policy_data_node_2]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_2]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_8_chunk 
+_timescaledb_internal._dist_hyper_1_11_chunk
+_timescaledb_internal._dist_hyper_1_14_chunk
+(3 rows)
+
+
+NOTICE:  [dist_policy_data_node_3]:  SELECT show_chunks('conditions')
+NOTICE:  [dist_policy_data_node_3]:
+show_chunks                                 
+--------------------------------------------
+_timescaledb_internal._dist_hyper_1_9_chunk 
+_timescaledb_internal._dist_hyper_1_12_chunk
+_timescaledb_internal._dist_hyper_1_15_chunk
+(3 rows)
+
+
+ remote_exec 
+-------------
+ 
+(1 row)
+
+SELECT remove_retention_policy('conditions');
+ remove_retention_policy 
+-------------------------
+ 
+(1 row)
+
+INSERT INTO conditions
```

Comment:
Is this maybe here by mistake? I don't see what the purpose of this INSERT would be.

Reply:
If you look at the bug report, there is an error when trying to insert after removing the retention policy, which is what we test that it works here.

Reply:
Ok, appreciate the extra comment with explanation